src/qtt/instrument_drivers/virtualAwg/awgs/ZurichInstrumentsHDAWG8.py

```diff
@@ -1,12 +1,19 @@
+import logging
+import os
 
+import numpy as np
+import zhinst
 from qcodes import Parameter
+
 from qtt.instrument_drivers.virtualAwg.awgs.common import AwgCommon, AwgCommonError
 
+logger = logging.getLogger(__name__)
+
 
 class ZurichInstrumentsHDAWG8(AwgCommon):
     __sampling_rate_map = {ii: 2.4e9 / 2**ii for ii in range(0, 14)}
 
-    def __init__(self, awg, awg_number=0):
+    def __init__(self, awg, awg_number=0, use_binary_waves=True):
```

Comment:
Are you sure we want the default behavior (binary instead of csv) to be different for all the users of  ZurichInstrumentsHDAWG8?

Reply:
There are no other users of the HDAWG8. The change is backwards compatible and should give a (modest) efficiency improvement.